app/controllers/concerns/course/forum/topic_controller_subscription_concern.rb

```diff
@@ -0,0 +1,37 @@
+module Course::Forum::TopicControllerSubscriptionConcern
+  extend ActiveSupport::Concern
+
+  def subscribe
+    if set_subscription_state
```

Comment:
Haven't found alias method in controller.

But think we can `skip_authorize_resource :subscribe` and manually `authorize :show` in subscribe action`

Another way is to define two sets of rules, but I think there are some duplications so I don't prefer it.